js/osselector.js

```diff
@@ -0,0 +1,14 @@
+var downloadLinks = document.querySelectorAll('.js-download');
+var os = 'linux';
+
+if (navigator.appVersion.indexOf('Win') != -1) { os = 'win-win' };
+if (navigator.appVersion.indexOf('MacOS') != -1) { os = 'mac' };
+if (navigator.appVersion.indexOf('Mac OS') != -1) { os = 'mac' };
+if (navigator.appVersion.indexOf('Macintosh') != -1) { os = 'mac' };
+
+for (var i = 0; i < downloadLinks.length; i++) {
```

Comment:
In general it is recommended to use `forEach` to loop on arrays, rather than the standard for loop.

```suggestion
downloadLinks.forEach(el => {
  if el.dataset.os === os) {
    el.classList.add('p-button--positive');
  }
});
```

Reply:
As there is no translilor its better to write ES5 so it works on more browsers.